README.md

```diff
@@ -31,9 +20,9 @@ Amplify SDK supports Android API level 16 (Android 4.1) and above.
 
 ### Using Gradle
 
-Gradle can be used alongside Android Studio to declare dependencies on Amplify right out of the box via Maven central repository. To use Amplify, simply specify the module dependencies inside app's `build.gradle` dependencies section.
+To use Amplify, specify which modules you want to use inside of your app's `build.gradle` dependencies section:
 
-``` gradle
+```gradle
 dependencies {
     // Core SDK is required for configuring Amplify
     implementation 'com.amplifyframework:core:0.10.0'
```

Comment:
Every plugin would already be dependent on core, so technically this would not be necessary. Not sure why I added it in the first place hmm

Reply:
will remove